packages/react-docs/RELEASE-NOTES.md

```diff
@@ -4,7 +4,7 @@ section: overview
 releaseNoteTOC: true
 ---
 
-## 2020.078 release notes (2020-06-24)
+## 2020.08 release notes (2020-06-24)
 - [@patternfly/react-catalog-view-extension@4.4.8](https://www.npmjs.com/package/@patternfly/react-catalog-view-extension/v/4.4.8)
 - [@patternfly/react-charts@6.5.4](https://www.npmjs.com/package/@patternfly/react-charts/v/6.5.4)
 - [@patternfly/react-core@4.23.1](https://www.npmjs.com/package/@patternfly/react-core/v/4.23.1)
```

Comment:
Can we bump all these versions to what was just released?
```
 - @patternfly/react-catalog-view-extension@4.4.8
 - @patternfly/react-charts@6.5.4
 - @patternfly/react-core@4.23.1
 - @patternfly/react-docs@5.4.11
 - @patternfly/react-icons@4.4.2
 - @patternfly/react-inline-edit-extension@4.5.7
 - demo-app-ts@4.17.5
 - @patternfly/react-styles@4.4.2
 - @patternfly/react-table@4.8.6
 - @patternfly/react-tokens@4.5.2
 - @patternfly/react-topology@4.4.7
 - @patternfly/react-virtualized-extension@4.4.8
```

Also topology may (or may not) be getting bumped for release by EOD.

Reply:
@redallen isn't that what I have?

Reply:
You've updated it since I left the comment. The versions are good now except for my latest review about topology!